### ISSUE TITLE
[Circle]: Use title case in strings

### ISSUE THREAD

**Issue by bragefuglseth** [SoongNoonien/plattenalbum#136]
Breakout thread from the ongoing [GNOME Circle review](https://gitlab.gnome.org/Teams/Circle/-/issues/219) of the app.

Per GNOME's [writing style guidelines](https://developer.gnome.org/hig/guidelines/writing-style.html), headings and short labels should use Title Case. The strings in the app should be updated to follow this.

**Comment by SoongNoonien** [comment]
I'd tried very hard to follow these guidelines. Where did I miss something?

**Comment by bragefuglseth** [comment]
Mostly the preferences view. It might not be obvious that the row titles should use title case, but they should :slightly_smiling_face: 

![Skjermbilde fra 2024-07-17 23-30-18](https://github.com/user-attachments/assets/27cd3dfd-a6cd-497a-83ac-06cccf259185)

**Comment by SoongNoonien** [comment]
Ok, I see. I thought this

> Sentence capitalization should be used for labels that form sentences or that run on to other text, including labels for check boxes, radio buttons, sliders, text entry boxes, field labels and combobox labels.

applies to the labels in the settings.

**Comment by bragefuglseth** [comment]
Those guidelines are indeed a little unclear. I'll look into improving them.

**Comment by SoongNoonien** [comment]
The preferences are now updated accordingly (4c3f49b8a06344c03d4797431117c7524892b92b). Are there any other mistakes you found, or can this be closed?

**Comment by bragefuglseth** [comment]
This can be closed, I’ll notify you if I ever find more strings where it’s needed! 😁
